app/lib/create_example_provider_users_with_permissions.rb

```diff
@@ -44,9 +44,12 @@ def self.call
       training_provider: Provider.find_by_code('1JB'),
       ratifying_provider: Provider.find_by_code('24P'),
     ).update(
-      training_provider_can_make_decisions: true,
-      training_provider_can_view_safeguarding_information: true,
-      training_provider_can_view_diversity_information: true,
+      ratifying_provider_can_make_decisions: [true, false].sample,
+      ratifying_provider_can_view_safeguarding_information: [true, false].sample,
+      ratifying_provider_can_view_diversity_information: [true, false].sample,
+      training_provider_can_make_decisions: [true, false].sample,
+      training_provider_can_view_safeguarding_information: [true, false].sample,
+      training_provider_can_view_diversity_information: [true, false].sample,
       setup_at: Time.zone.now,
```

Comment:
Can we use a FactoryBot to achieve this instead, similar to https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/lib/test_applications.rb? There you can use Faker to retrieve a random value for each, using `Faker::Boolean.boolean(true_ratio: 0.5)`

Reply:
Sure thing, happy to chat